### ISSUE TITLE
Pagination seem to not work

### ISSUE THREAD

**Issue by kzonov** [cdragon1116/squarespace_api#1]
Hi there,
Thanks a bunch for taking the effort to implement this gem. A huge bummer that there is no official one for Squarespace.
I stumbled upon one issue, when it comes to pagination, the code doesn't work:
```
orders = []
    orders = client.orders.all do |orders_per_page|
      orders += orders_per_page
    end
```
Results in:
```
.asdf/installs/ruby/3.1.0/lib/ruby/gems/3.1.0/gems/squarespace_api-0.0.6/lib/squarespace_api/paginated_fetch.rb:7:in `dig': no implicit conversion of String into Integer (TypeError)
```

I guess it happens because here happens the `parse_collection` https://github.com/cdragon1116/squarespace_api/blob/master/lib/squarespace_api/resource_group_actions.rb#L9
and for `orders` it removes the metadata part, leaving only the list of orders https://github.com/cdragon1116/squarespace_api/blob/master/lib/squarespace_api/resource_groups/orders.rb#L13

If I'm doing smth wrong - please let me know, otherwise I guess I'll prepare a fix

**Comment by cdragon1116** [comment]
@graffzon  thanks for spotting. I will check the PR ASAP.

**Comment by cdragon1116** [comment]
Thanks for your help. 
The changes were included in the 0.0.7 release.

https://github.com/cdragon1116/squarespace_api/releases/tag/v0.0.7

https://rubygems.org/gems/squarespace_api/versions/0.0.7
